blue/dev-lab.sh

```
@@ -0,0 +1,156 @@
+#
+# 這支存在的理由是「一個人一台筆電要把整套跑起來看看」—— 那個情境下你同時扮演
+# 主辦方與來賓，於是需要有人把兩邊的步驟接起來。
+#
+# 用法：
+#   ./blue/dev-lab.sh              起靶機 ＋ 裝 AI
+#   ./blue/dev-lab.sh --no-ai      只起靶機
+#   ./blue/dev-lab.sh down         收掉（容器與網路，不動 image 與 AI）
+#   LAB_ID=02 PORT=18081 ./blue/dev-lab.sh    同一台跑第二套
+set -uo pipefail
+
+cd "$(dirname "$0")/.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)